tests/snuba/api/endpoints/test_organization_stats_v2.py

```diff
@@ -512,21 +508,51 @@ def test_org_group_by_project(self):
         assert result_sorted(response.data) == {
             "start": "2021-03-14T00:00:00Z",
             "end": "2021-03-14T12:28:00Z",
-            "intervals": ["2021-03-14T00:00:00Z"],
             "groups": [
                 {
                     "by": {"project": self.project.id},
                     "totals": {"sum(times_seen)": 6},
-                    "series": {"sum(times_seen)": [6]},
                 },
                 {
                     "by": {"project": self.project2.id},
                     "totals": {"sum(times_seen)": 1},
-                    "series": {"sum(times_seen)": [1]},
                 },
             ],
         }
 
+    @freeze_time("2021-03-14T12:27:28.303Z")
+    def test_org_project_totals_per_project(self):
```

Comment:
question: does this test fail before your changes? might be worth running it on the master branch to see. you may need to add an outcome with a timestamp in the period of the previously out-of-bounds interval for this to truly test the behavior.